llvm/docs/NVPTXUsage.rst

```diff
@@ -568,6 +568,99 @@ to left-shift the found bit into the most-significant bit position, otherwise
 the result is the shift amount needed to right-shift the found bit into the
 least-significant bit position. 0xffffffff is returned if no 1 bit is found.
 
+'``llvm.nvvm.zext.inreg.clamp``' Intrinsic
```

Comment:
Do we need the `inreg` part? 

It sounds like an implementation detail at best, and misleading, at worst. I.e. I'd assume that it implies in-place (i.e. in the same register) conversion, which is not the case.

Reply:
I'll remove it. The intent was to match the convention of the `ISD::SEXT_INREG` node which performs an operation similar to this if `%b` were a constant.